tfgrid3/ubuntu22.04/fullvm/README.md

```diff
@@ -69,4 +69,6 @@ tar -czf ubuntu-jammy.tar.gz -C ubuntu-jammy .
 clsecret="$API_KEY"
 curl -X Post -H "Authorization: Bearer ${clsecret}" -F "file=@ubuntu-jammy.tar.gz"  https://hub.grid.tf/api/flist/me/upload
```

Comment:
could you please use the version number in the name of the flist better than using image codename to be more clear for users when adding it to hub then dashboard

Reply:
pushed docker images